pack.toml

```diff
@@ -2,9 +2,16 @@
 type = "local"
 path = "."
 ipkg = "http.ipkg"
-test = "test/test.ipkg"
+test = "tests/http-test.ipkg"
+
+[custom.all.tls]
```

Comment:
its all kind of messy, but we need the tls so file in tests exec folder ... is this the correct way? why do we need this?

Reply:
I thought the fix in the tls package would solve this issue. Propably not

Reply:
maybe i should try to clear pack cache or something, and try build again